app/views/users/show.html.erb

```diff
@@ -1 +1,14 @@
-<h1>User Profile</h1>
+<main>
+  <section>
+    <ul>
+      <% @user.recent_posts.each do |post| %>
+        <%= link_to user_user_post_path(@user, post) do %>
+          <li class="card flexColumn hover">
+            <h3 class="left"><%= post.title %></h3>
+            <h4 class="left"><%= post.text %></h4>
+          </li>
+        <% end %>
+      <% end %>
```

Comment:
- [ ] According to the wireframe the users#show page should show the users' photo, name, bio, and postCounter, please 🙏🏽 add this to show the most recent posts. After that kindly add a "See all posts" button, to allow the users to see all posts. 

![image](https://github.com/MarcoDDM/blog-app/assets/58642949/2c39c8e7-4b09-4794-ba41-b74d91b49583)